src/main/java/io/fabric8/jenkins/openshiftsync/ConfigMapWatcher.java

```diff
@@ -190,6 +196,46 @@ private boolean containsSlave(ConfigMap configMap) {
                 podTemplate = xStream2.fromXML(entry.getValue());
 
                 if( podTemplate instanceof PodTemplate ) {
+                    PodTemplate pt = (PodTemplate)podTemplate;
+                    
+                    // if the image field is and imagestream related ref, without the internal docker registry, prepend it
+                    String image = pt.getImage();
+                    try {
+                        if (!OpenShiftUtils.hasRegistry(image)) {
```

Comment:
so not a new xml tag, just more intelligent processing of the value of the existing tag (similar to what you were doing, except making the user declare their intention more explicitly)